map-widget/qmlmapwidgethelper.cpp

```diff
@@ -162,7 +162,7 @@ void MapWidgetHelper::selectedLocationChanged(struct dive_site *ds_in)
 void MapWidgetHelper::selectVisibleLocations()
 {
 	QList<int> selectedDiveIds;
-	for (auto [idx, dive]: enumerated_range(divelog.dives)) {
+	for (auto [idx, dive] : enumerated_range(divelog.dives)) {
```

Comment:
Please don't do gratuitous white space changes. In particular when they are inconsistent with the rest of the code base.